### ISSUE TITLE
Kodi (LibreElec) crash and restart when I try to play a video

### ISSUE THREAD

**Issue by nonobio** [CastagnaIT/plugin.video.netflix#1497]
### Netflix add-on version

v1.20.1+matrix.1

### Operative systems used

LibreELEC

### Kodi version used

Kodi 19 (Matrix)

### Description of the bug

Hi,
When I try to play a video from the Netflix-addon, my Kodi LibreElec restart without any error displayed
Thanks :)

### Steps to reproduce the behavior

1. Open Netflix-addon
2. Choose my profile
3. Open "my list"
4. Choose a show
5. Choose a season
6. Press ok on an episode
7. Kodi LibreElec crash and restart

### Debug log - mandatory

log : https://paste.kodi.tv/nenefoxojo.kodi crash log : https://paste.kodi.tv/sojugurapi.kodi

### Additional context

I have another similar device (PC) with the same LibreElec, Kodi, and Netflix-addon version and it hasn't the issue

**Comment by CastagnaIT** [comment]
there is some kind of problem with the widevine library
these errors seem to say that is not able to load the library

```
2022-11-10 10:08:30.463 T:1175    DEBUG <general>: [script.module.inputstreamhelper] Found Widevine CDM at /storage/.kodi/cdm/libwidevinecdm.so
2022-11-10 10:08:30.551 T:1175    FATAL <general>: [script.module.inputstreamhelper] ['ldd', '/storage/.kodi/cdm/libwidevinecdm.so'] cmd failed.
2022-11-10 10:08:30.552 T:1175    DEBUG <general>: [script.module.inputstreamhelper] ['ldd', '/storage/.kodi/cdm/libwidevinecdm.so'] cmd output:
                                                   Bus error (core dumped)
                                                   ldd: $exited with unknown exit code (135)
                                                   
2022-11-10 10:08:30.552 T:1175    FATAL <general>: [script.module.inputstreamhelper] Failed to check for missing Widevine libraries.
```
```
#11 0x00007f259a3ec326 in dlopen () from /usr/lib/libdl.so.2
#12 0x00007f255f83ffc1 in base::LoadNativeLibrary(std::string const&, base::NativeLibraryLoadError*) () from /storage/.kodi/addons/inputstream.adaptive/libssd_wv.so
```

its possible that script.module.inputstreamhelper addon has installed or updated with a wrong widevine version
you have to ask for help and open a report to:
https://github.com/emilsvennesson/script.module.inputstreamhelper

**Comment by nonobio** [comment]
Thanks @CastagnaIT , I created an issue [there](https://github.com/emilsvennesson/script.module.inputstreamhelper/issues/524).

**Comment by nonobio** [comment]
Hi,

I checked versions on my other device where it works:

-Netflix addon settings
-Expert tab
-InputStream Helper addon settings
-Expert tab
-InputStream Helper information

InputStream Helper: 0.5.10+matrix.1
InputStream Adaptive: 19.0.7.1
Widevine CDM: 4.10.2449.0
Installé dans /Storage/.kodi/cdm

All versions are the same that on my device where I have the issue...

So, on the same place, I tried to uninstall Widevine CDM library, then reinstall it.
Reinstallation failed several times but finally it worked (I had to accept agreements...)

Then video playing worked!

I only watched one video, but I hope it will work for others videos :).
